float32 Dist_obj0
float32 Dist_obj1
float32 Dist_obj2
float32 Dist_obj3
float32 Dist_obj4
float32 Dist_obj5

bool Detect2m_obj0 
bool Detect2m_obj1 
bool Detect2m_obj2
bool Detect2m_obj3
bool Detect2m_obj4
bool Detect2m_obj5


